src/CentileChart/CentileChart.tsx

```diff
@@ -116,7 +117,6 @@ function CentileChart({
         [storedChildMeasurements, sex, measurementMethod, reference, showCorrectedAge, showChronologicalAge],
     );
 
-
     const updatedData = useMemo(() => getVisibleData(sex, measurementMethod, reference, userDomains), [
```

Comment:
Can we remove this? It doesn't seem used anywhere

Reply:
I have removed. let's see if that is ok.